src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardTokens.tsx

```diff
@@ -156,14 +160,14 @@ const IfoCardTokens: React.FC<IfoCardTokensProps> = ({
       )
     }
     if (publicIfoData.status === 'finished') {
-      return userPoolCharacteristics.amountTokenCommittedInLP.isEqualTo(0) ? (
+      return userPoolCharacteristics.amountTokenCommittedInLP.isEqualTo(1) ? (
```

Comment:
It should be 0 no ?

Reply:
ah good catch. it's just for debug lol 🙏

Reply:
Should we use constant for a value ? Just make the number meaningful

Reply:
This line is pretty self explain: *amount token committed in LP is equal to 0*
I will leave it for now, thanks for the suggestion